### ISSUE TITLE
MD034 flags URLs in front matter

### ISSUE THREAD

**Issue by zane** [rvben/rumdl#116]
## Reproduction steps

```bash
echo '---\nurl: http://example.com\n---\n\n# Repro' | rumdl check -
```

## Expected results

```
No issues found in <stdin>
```

## Actual results

```
<stdin>:2:6: [MD034] URL without angle brackets or link formatting: 'http://example.com' [*]
```

**Comment by rvben** [comment]
Hi @zane,

Thanks for the clear issue and clear reproduction steps. Fix for this will land in the upcoming release.

**Comment by rvben** [comment]
Hi @zane,

Now we're skipping front matter for every markdown rule, as it is never markdown, but toml,json or yaml.

Thanks for pointing me to this and making rumdl better for everyone :) 👍
